tokio/src/lib.rs

```diff
@@ -483,7 +483,7 @@ pub(crate) use self::doc::winapi;
 
 #[cfg(all(not(docsrs), windows, feature = "net"))]
 #[allow(unused)]
-pub(crate) use ::winapi;
+pub(crate) use winapi;
```

Comment:
what is this change?

Reply:
I think rustfmt did it? I didn't manually change it but I don't think it affects anything since this got updated in 2018 edition iirc